test/ci/azure/cpu-mac.yml

```diff
@@ -6,32 +6,54 @@
     name: openvinotoolkit/openvino_tensorflow
     
  pool:
-  name: 'macOS'
+  vmImage: 'macOS-10.15'
   
  variables:
     WORK_DIR: $(Pipeline.Workspace)/openvino_tensorflow
-    OV_LOCATION: /Users/intel/ci_setup/openvino_2022.1_abi0/
+    OV_LOCATION: $(Pipeline.Workspace)/openvino/
     
  steps:
  
-   - script: |
-      rm -rf $(WORK_DIR) ; mkdir $(WORK_DIR)
+   # - script: |
+   #    rm -rf $(WORK_DIR) ; mkdir $(WORK_DIR)
+   - task: UsePythonVersion@0
+     inputs:
+      versionSpec: '3.8'
+      addToPath: true
+      #architecture: 'x64' # Options: x86, x64 (this argument applies only on Windows agents)
    
    - checkout: self
      clean: true
      lfs: false
      path: openvino_tensorflow
+
+   - script: |
+      wget https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.1/m_openvino_toolkit_runtime_p_2022.1.0.643.tgz
+      mkdir -p ${OV_LOCATION}
+      tar -xvf m_openvino_toolkit_runtime_p_2022.1.0.643.tgz -C ${OV_LOCATION}
+     displayName: "Install OpenVINO" 
  
    - script: |
       python3 -m pip install -r requirements.txt
+      pip3 install -U pytest
      displayName: "Setup" 
     
    - script: |
       git submodule init
       git submodule update
-      python3 build_ovtf.py --use_openvino_from_location=${OV_LOCATION} --cxx11_abi_version=0
+      python3 build_ovtf.py --use_openvino_from_location=${OV_LOCATION}/m_openvino_toolkit_runtime_p_2022.1.0.643/ --cxx11_abi_version=0
      workingDirectory: $(WORK_DIR) 
      displayName: "Build"
+
+   - script: |
+      echo "---------Load Libraries before changing name---------"
```

Comment:
Possible to use relative paths? What if path changes to ../work/2 instead?

Reply:
removed this step in latest changes, since issue is resolved by adding CMAKE_BUILD_TYPE=Release flag.